aha-table.html

```diff
@@ -429,15 +429,23 @@
       var scrollBody = Polymer.dom(aha.root).querySelector('.scroll-body.aha-table');
       var rowHeight = 42;
 
+      var closestPageSize;
```

Comment:
But I would like to keep the event listener. And it will only fire one time when the page load. "_isAttached" function also attached, but there is no detached for that function. So let me know when to detached for that function and why it's necessary for the function.